mac/Keyman4MacIM/Keyman4MacIM/KMDownloadKeyboard/KMDownloadKBWindowController.m

```diff
@@ -23,7 +23,11 @@ - (void)windowDidLoad {
     [super windowDidLoad];
     
     [self.webView setFrameLoadDelegate:(id<WebFrameLoadDelegate>)self];
-    [self.webView.mainFrame loadRequest:[NSURLRequest requestWithURL:[NSURL URLWithString:@"https://secure.tavultesoft.com/prog/macosx/10/downloadkeyboards/"]]];
+    
+    NSString *version = [[[NSBundle mainBundle] infoDictionary] objectForKey:@"CFBundleShortVersionString"];
+    NSString *url = [NSString stringWithFormat:@"https://keyman.com/go/macos/10.0/download-keyboards/?version=%@", version];
```

Comment:
There is a "URL types" entry in the Info.plist that has a URL identifier that still references tavultesoft. I'll probably need to consult the history, but I think maybe this is the thing we had to add to allow access to tavultesoft.com (it's not the way I remembered it, so I'm a little uncertain). I think we need to understand what it is and figure out how it relates to the URL we point to for the downloads, so we can figure out how/if it needs to be changed or if it can maybe be removed altogether.

Reply:
Okay. I suspect that will relate to the downloading of the keyboard rather than the presentation of the web page within the browser? It will certainly need to change if that's the case -- the new download server is https://downloads.keyman.com

Reply:
No, if I'm right (which I might not be) it was a bug that was causing the download page to display blank. (But I just tried in vain to find such an issue.) I believe it was the very first bug I worked on last summer.

Reply:
Okay, it's not that. I finally tracked down that old issue (in the archived private repo). I still need to figure out what the "URL types" entry is, but I don't think that needs to hold this up.